test/e2e/storage/csi_mock/csi_volume_expansion.go

```diff
@@ -274,6 +274,11 @@ var _ = utils.SIGDescribe("CSI Mock volume expansion", func() {
 				err = e2epod.WaitForPodNameRunningInNamespace(ctx, m.cs, pod.Name, pod.Namespace)
 				framework.ExpectNoError(err, "Failed to start pod1: %v", err)
 
+				pvc, err = m.cs.CoreV1().PersistentVolumeClaims(pvc.Namespace).Get(ctx, pvc.Name, metav1.GetOptions{})
+				if err != nil {
+					framework.Failf("failed to get pvc %s, %v", pvc.Name, err)
+				}
+				gomega.Expect(pvc.Spec.VolumeName).ShouldNot(gomega.BeEquivalentTo(""), "while provisioning a volume for resizing")
 				pv, err := m.cs.CoreV1().PersistentVolumes().Get(ctx, pvc.Spec.VolumeName, metav1.GetOptions{})
```

Comment:
To try to debug the test some more, we can also add validation that the `PV.spec.csi.nodeExpandSecretRef` is set correctly.

Reply:
Added:
https://github.com/kubernetes/kubernetes/pull/116568/files#diff-6418fc90a4ed7e7c3cbfe9119ad04e252cf38d9ff1be39fcde2eb7f41178d641R289-R291